src/absil/ilwrite.fs

```diff
@@ -4025,17 +4037,18 @@ let writeBinaryAndReportMappings (outfile, ilg, pdbfile: string option, signer:
           let entrypointCodeChunk,next = chunk 0x06 next
           let globalpointerCodeChunk,next = chunk (if isItanium then 0x8 else 0x0) next
           
-          let debugDirectoryChunk,next = chunk (if pdbfile = None then 0x0 else sizeof_IMAGE_DEBUG_DIRECTORY) next
+          let debugDirectoryChunk,next = chunk (if pdbfile = false then 0x0 else sizeof_IMAGE_DEBUG_DIRECTORY) next
```

Comment:
@dsyme , sry, my bad, when there is no comment, is ok for me to change it. I'll always write reply next time